index.bs

```diff
@@ -789,23 +864,41 @@ with parameter |value| is:
 
 <div algorithm="webSocketUrl new session algorithm">
 The [=WebDriver new session algorithm=] defined by this specification,
-with parameters |session| and |capabilities| is:
+with parameters |session|, |capabilities|, and |flags| is:
+
+1. If |flags| contains "<code>bidi</code>", return.
 
- 1. Let |webSocketUrl| be the result of [=getting a property=] named
-    "<code>webSocketUrl</code>" from |capabilities|.
+1. Let |webSocketUrl| be the result of [=getting a property=] named
+   "<code>webSocketUrl</code>" from |capabilities|.
 
- 2. If |webSocketUrl| is undefined, return.
+1. If |webSocketUrl| is undefined, or false, return.
```

Comment:
I don't have a strong position on the Oxford comma, but this comma made it harder to read IMHO:

```suggestion
1. If |webSocketUrl| is undefined or false, return.
```